.gitignore

```diff
@@ -8,9 +8,11 @@ umoci
 skopeo
 
 # Binaries
-agent-protocol-forwarder
-cloud-api-adaptor
-kata-agent
+# The leading `/` avoids recursive matching thereby ensuring
+# the code files under cmd/cloud-api-adaptor are not ignored
+/agent-protocol-forwarder
+/cloud-api-adaptor
+/kata-agent
```

Comment:
Do we need to change these lines as suggested at https://github.com/confidential-containers/cloud-api-adaptor/issues/578#issuecomment-1413422859?

Reply:
Added